### DISCUSSION TITLE
middleware/first-visit/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1862] [Announcements]
# middleware/first-visit/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/first-visit/

**Comment by liyuerich** [comment]
前提条件可以稍微说一下，毕竟要部署在集群上，只写workspace的准备是否不够？

**Reply by Michelle951** [reply]
fixed by pull request: https://github.com/DaoCloud/DaoCloud-docs/pull/1885
